pcer/experiment.py

```diff
@@ -12,14 +13,20 @@ def __init__(self):
         self.session = Session('db.json', 'experiment')
         pass
 
+    #Added new function setCurrent participant for modularity
+    def setCurrentParticipant(self,p_id, p_group=None):
+        self.participant_id = p_id
+        self.participant_group = p_group
+
     def addParticipant(self, p_id, p_group):
         print("Experiment.openParticipanSession")
+        self.setCurrentParticipant(p_id, p_group)
```

Comment:
the `participant_id` and `participant_group` are supposed to be for the current participant. If the id is `None`, it means there is no active participant. I think that this should avoid the use the 'loaded_id' flag, which I prefer avoid using it.

Reply:
For the second case mentioned in the issue #12  - "If ID is different", 
Only way to check it was parsing the QPlainTextEdit widget and look for participant_id among all other fields in widget. It is a single string. I think instead we can rely on internal variable (loaded_id) which tells us what is in QPlainTextEdit. @hvaldecantos